app/utils/press-release-snapshot.js

```diff
@@ -53,6 +56,17 @@ export default class PressReleaseSnapshot {
     this.pressRelease.governmentFields = this.governmentFields;
     this.pressRelease.themes = this.themes;
     this.pressRelease.sources = this.sources;
+
+    //destroys and record which is added but not saved
+    if (this.pressRelease.attachments.length) {
+      this.pressRelease.attachments.forEach(attachment => {
+        const isNewlyAdded = this.attachments.indexOf(attachment) === -1;
```

Comment:
Ter info: er bestaat ook `myArray.includes` in JS (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)